js/modules/circles.js

```diff
@@ -6,17 +6,18 @@ function add_circles(dv_stats_data, scale_colors_fxn) {
     .enter()
     .append("circle")
       .classed("gage_point", true)
-      .attr("cx", function(d, i) { return i*10 + 20; })
-      .attr("cy", function(d, i) { return Math.random()*200; })
-      .attr("r", function(d) { return d.per*10; })
-      .attr("fill", function(d) { return scale_colors_fxn(d.per); })
+      .attr("cx", function(d) { return d.x; })
```

Comment:
Looks good, but how is the performance w/ circles? I am assuming drawing them this way is temporary and we'd use canvas or path later?

Reply:
Yes, this way is temporary because I assume we will use canvas later. I did this while canvas was still a maybe and we wanted to continue working on the interactions. Performance is not actually bad (at least on my localhost)